core/templates/dev/head/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -142,6 +144,12 @@ angular.module('oppia').directive('adminMiscTab', [
           ctrl.dataExtractionQueryStatusMessage = message;
         };
 
+
+        ctrl.dummyMail = function() {
+          var SendMailUrl = SEND_DUMMY_MAIL_HANDLER_URL;
```

Comment:
Variable names should be lowercased. (Please follow the format of surrounding code.)

Reply:
Done